### DISCUSSION TITLE
About security.js

### DISCUSSION THREAD

**Discussion by artmadeit** [mercadopago/sdk-js#145] [Q&A] [answered]
Hey guys, I was checking this link on the docs: https://www.mercadopago.com.pe/developers/es/docs/checkout-bricks/how-tos/improve-payment-approval, which is referenced on the "requisites to go to production" page

As the link says, I added the script tag
`<script src="https://www.mercadopago.com/v2/security.js" view="checkout"></script>`

However when I access, the deviceId, I'm always getting undefined

![image](https://user-images.githubusercontent.com/18428325/227792479-20f7d5bb-59fd-47a3-9125-b65da2fd7cef.png)

Also other question: Is that script security needed on when using Checkout bricks? And how is the flow to integrate with python (or any backend language)

**Comment by icaldana** [comment, marked as answer]
Hello @artmadeit!

I double-checked, and I was able to see the values properly  🤔 

![image](https://user-images.githubusercontent.com/30550706/228057035-5bcc7a22-07f9-409b-a87f-983d561fad6d.png)

1. Could you please provide more details or a code snippet? 
2. About the security script on Bricks, it is not required, although it is nice to have it. Here is an example of how this looks like on python

```
import mercadopago
from mercadopago.config import RequestOptions

request_options = RequestOptions(custom_headers={"X-meli-session-id": "<replace_here>"})
# ...

result = sdk.payment().create(payment_data, request_options)
```

I hope that helps, thank you!
